tests/test_issue_1514/test_rp_integration.py

```diff
@@ -0,0 +1,201 @@
+                          'action': rp.TRANSFER}
+    cuds.append(cud)
+
+    ## Test for multiple folder
+    cud = rp.ComputeUnitDescription()
+    cud.executable     = '/bin/date'
+    cud.input_staging  = {'source':os.path.join(local_sample_data, sample_data[2]), 
+                          'target':'unit:///%s' % sample_data[2],
+                          'action': rp.TRANSFER} 
+
+    cud.output_staging = {'source':'unit:///%s' % sample_data[2], 
+                          'target': './multi_folder_2',
+                          'action': rp.TRANSFER}                
+
+    cuds.append(cud)
+
+    ## Submit the previously created ComputeUnit descriptions to the
+    ## PilotManager. This will trigger the selected scheduler to start
+    ## assigning ComputeUnits to the ComputePilots.
+    units = umgr.submit_units(cuds)
+
+    ## Wait for all compute units to reach a final state (DONE, CANCELED or FAILED).
+    umgr.wait_units()
+    # ------------------------------------------------------------------------------------------------------------------
+  
+    # Extract info from RP config file
+    #-------------------------------------------------------------------------------------------------------------------
+    config_loc = '../../src/radical/pilot/configs/resource_%s.json'%resource.split('.')[0]
+    path_to_rp_config_file = os.path.realpath(os.path.join(os.getcwd(),config_loc))
+    cfg_file = ru.read_json(path_to_rp_config_file)[resource.split('.')[1]]
+    access_schema = config[resource]['schema']
+    if not access_schema:
+        access_schema = 'ssh'
+
+
+    # Verify the actionables were done for single file transfer
+    # ------------------------------------------------------------------------------------------------------------------
+    remote_dir = rs.filesystem.Directory(   units[0].sandbox, 
+                                            session=session)
+
+    ## Test on remote
+    assert sample_data[0] in [x.path for x in remote_dir.list()]
+    ## Test on client
+    local_file = sample_data[0].split('.')[0] + '_2.' + sample_data[0].split('.')[1]
+    assert local_file in [os.path.basename(x) for x in glob('./*')]
+    # ------------------------------------------------------------------------------------------------------------------
+
+
+    # Verify the actionables were done for single folder transfer
+    # ------------------------------------------------------------------------------------------------------------------
+    remote_dir = rs.filesystem.Directory(   units[1].sandbox, 
+                                            session=session) 
+    ## Test on remote
+    assert sample_data[1] in [x.path for x in remote_dir.list()]
+    for x in remote_dir.list():
+        if remote_dir.is_dir(x):
+            child_x_dir = rs.filesystem.Directory(os.path.join(units[1].sandbox, x.path) ,
+                                                    session=session)
```

Comment:
you sometimes use `/`, and sometimes `os.path.join()`.  We don't need the method really, since we won't run on Windows anyway...

Reply:
Makes sense. Either way I need to stick to one mode.